compiler/rustc_middle/src/mir/visit.rs

```diff
@@ -1223,6 +1223,14 @@ impl PlaceContext {
         }
     }
 
+    /// Returns `true` if `self` describes a move out of the place.
+    pub fn is_move(&self) -> bool {
+        match *self {
+            PlaceContext::NonMutatingUse(NonMutatingUseContext::Move) => true,
+            _ => false,
+        }
```

Comment:
```suggestion
        matches!(self, PlaceContext::NonMutatingUse(NonMutatingUseContext::Move))
```
This prompted me to submitted a PR adapting the surrounding code as well.